src/tcgwars/logic/impl/gen8/RebelClash.groovy

```diff
@@ -2029,10 +2031,10 @@ public enum RebelClash implements LogicCardInfo {
         resistance F, MINUS30
         bwAbility "Counterattack", {
           text "If this Pokemon is your Active Pokemon and is damaged by an opponent’s attack, place 3 damage counters on the attacking Pokemon."
-          delayedA {
+          delayedA (priority: LAST) {
             before APPLY_ATTACK_DAMAGES, {
               if (bg.currentTurn == self.owner.opposite && bg.dm().find({ it.to==self && it.dmg.value }) && self.active) {
-                powerused()
+                bc "Counterattack activates"
```

Comment:
Why this change? As long as the proper method `powerUsed()` is used, it should provide the same output. The issue is that `powerused()` doesn't exist.

Reply:
powerUsed doesn't seem to be used on any passive abilities? Removed it so it works exactly like other "if damaged, put 3 counters on..." abilities, like on Dragon Majesty

Reply:
I suppose. I had added this just because there was no text displayed at all. I figured it'd be easier to use `powerUsed` in any ability that you just need to know it activated and that's what caused additional things in the game log to happen. Either way it works I guess, but I'd at least include the Pokémon's name.